paper.tex

```diff
@@ -1972,8 +1982,43 @@ \section{Conclusion and Future Work}
 Kinkaku-ji. More extensive user testing at the actual site could have helped us
 catch bugs earlier and lead to a more polished final product.
 
-Despite these issues our team is happy with the product we delivered to
-Kyoto~VR. Selling ARuko on app stores was not within the scope of our academic
+While not a disaster, we had some initial trouble getting started with Unity.
+Different versions of Unity do not play well with each other, and it often runs
+into bugs on older hardware, such as the laptops we had to work with.
+Furthermore, a stable version of the Unity editor is not available on
+GNU/Linux~\autocite{best2019}, so one of our team members had to install MacOS
```

Comment:
pretty sure officially, it is capitalized macOS, even according to Wikipedia

Reply:
Fixed